src/main/java/org/kainos/ea/DropwizardApplicationConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package org.kainos.ea;
+
+import io.dropwizard.Configuration;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.federecio.dropwizard.swagger.SwaggerBundleConfiguration;
+import org.hibernate.validator.constraints.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+
+public class DropwizardApplicationConfiguration extends Configuration {
+    // TODO: implement service configuration
+    @Valid
+    @NotNull
+    private final SwaggerBundleConfiguration swagger = new SwaggerBundleConfiguration();
+
+    @JsonProperty("swagger")
```

Comment:
Swagger is registered in the project, but doesn't have the annotations on the methods. Would it be worth removing the swagger integration until there is an endpoint that isn't for test usage being used?